src/widget/taskBar.js

```diff
@@ -100,6 +100,9 @@ var TaskBar = GObject.registerClass(
                         item.connect('close-clicked', (_) => {
                             tileable.kill();
                         });
+                        item.tileable.connect('disconnect-tileable', (_) => {
```

Comment:
I'm not sure to understand here. The tileable is gonna be destroyed and all signal bind to it will also be removed. The only things we want to make sure is to not have reference to the signal anymore when it's destroyed and I guess with the new check on the MsManager to remove all reference to signal if it's a Clutter.Actor on Destroy should be good enough I believe.
But maybe I miss something

Reply:
Sometimes the window is killed before the tileableitem is, when waiting for windows on launch, for example, and this results in trying to remove signals for a window that does not exist anymore...

Reply:
The metaWindow can be destroyed before but not the tileable item. I think I missunderstand something here